src/CpmsClient/Data/AccessToken.php

```diff
@@ -1,123 +1,89 @@
 <?php
+
 namespace CpmsClient\Data;
 
 use Laminas\Stdlib\AbstractOptions;
+use Traversable;
 
 /**
  * Class AccessToken
+ * @extends AbstractOptions<iterable>
  *
  * @package CpmsClient\Data
  */
 class AccessToken extends AbstractOptions
 {
-    const INVALID_ACCESS_TOKEN = 114;
-    /**
-     * @var string
-     */
-    protected $expiresIn;
-    /** @var  string */
-    protected $tokenType;
-    /** @var  string */
-    protected $accessToken;
-    /** @var  string */
-    protected $scope;
-    /** @var  int */
-    protected $issuedAt;
-    /** @var  string */
-    protected $salesReference;
-
-    public function __construct($options = null)
+    public const INVALID_ACCESS_TOKEN = 114;
+    protected ?int $expiresIn = null;
+    protected ?string $tokenType = null;
+    protected ?string $accessToken = null;
+    protected ?string $scope = null;
+    protected ?int $issuedAt = null;
+    protected ?string $salesReference = null;
+
+    public function __construct(?iterable $options = null)
     {
         $this->__strictMode__ = false;
+        /**
+         * @phpstan-ignore-next-line
```

Comment:
Can we limit this to just the rule that's failing? So we don't accidentally mask something else